examples/packer-aws/ubuntu-2204.pkr.hcl

```diff
@@ -0,0 +1,82 @@
+packer {
+  required_plugins {
+    amazon = {
+      version = ">= 1.1.0"
+      source  = "github.com/hashicorp/amazon"
+    }
+    mondoo = {
+      version = ">= 0.2.1"
```

Comment:
I'd suggest >0.3.0 as we already have a new minor out

Reply:
Agreed!